README.md

```diff
@@ -345,3 +375,4 @@ jobs:
           apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
           accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
 ```
+````
```

Comment:
Looks like the `README.md` markdown isn't rendering correctly, maybe because of this stray ````?

<img width="1132" alt="image" src="https://github.com/user-attachments/assets/82589af8-4650-4fec-8b2e-0776e236ecdb">

Reply:
Thanks! Fixed :)